src/services/mongo.ts

```diff
@@ -43,7 +43,8 @@ const connect = () =>
         reject(err);
       } else {
         console.log(`Connected to mongo at mongodb://${mongoHost}/${mongoDb}`);
-        resolve();
+        resolve(null); // null required by 'strictNullChecks'
```

Comment:
TypeScript guys realised they walked themselves into a corner... still trying to figure their own way out of it.
https://github.com/microsoft/TypeScript/issues/29131